source/technical/shares.rst

```diff
@@ -0,0 +1,234 @@
+.. _shares:
+
+====================
+Shares
+====================
+
+Chameleon provides a shared file system service through the `OpenStack Manila <https://docs.openstack.org/manila/latest/>`_ interface.
+With the service, you can create a shared file system, mount to the bare metal instances, and manage some of its properties, such as visibility.
+
+  .. hint::
+  
+    Chameleon shared file system service is currently backed by a CephFS. Same as our :ref:`object store <object-store>` service, the data is
+    replicated and the replication should provide good availability in case of hardware failures.
+
+    *Difference between shared file system and object store*
+	You can choose either shared file system or object store to store, manage, and share your data with your collaborators. The object store
+	is suitable for storing large objects at scale, but isn’t suitable for transactional data, as objects are immutable and updated in their
+	entirety. On the other hand, Chameleon shared file system maps the CephFS using file-based NFS protocol to the bare metal instances, and
+	the NFS protocol manages locking and data integrity processes required to provide multiple concurrent access to data. 
```

Comment:
```suggestion
	entirety. Chameleon shared file system instead provides a NFS mount to the bare metal instances, with
	the NFS protocol managing locking and data integrity processes required to provide multiple concurrent access to data.
```

Reply:
Kind of a nit, "on the other hand" implies the sentence is going to have the same structure (describing access patterns and how writes work), but it describe slightly different aspects of the solution.